osu.Game/Overlays/Chat/DrawableUsername.cs

```diff
@@ -161,14 +163,22 @@ public MenuItem[] ContextMenuItems
                 if (!user.Equals(api.LocalUser.Value))
                     items.Add(new OsuMenuItem(UsersStrings.CardSendMessage, MenuItemType.Standard, openUserChannel));
 
-                if (currentChannel?.Value != null)
+                if (currentChannel?.Value != null && (!user.Equals(api.LocalUser.Value)))
                 {
                     items.Add(new OsuMenuItem(ChatStrings.MentionUser, MenuItemType.Standard, () =>
                     {
-                        currentChannel.Value.TextBoxMessage.Value += $"@{user.Username} ";
+                        currentChannel.Value.TextBoxMessage.Value += $"@{username} ";
                     }));
                 }
```

Comment:
When I implemented this, I made mentioning yourself via context menu intentional for two reasons:
- Discord does this (except on dms)
- to set a note for yourself

Needs more opinions.

Reply:
The way that I see it, the only reason to mention yourself on discord is if you are saying something from a previous time in the conversation WITH the intention of that mention having the text either readable in the mention line or being a "link" to the message. At this point in lazer neither of those is true, so mentioning yourself just leads to... nothing but your username with an @? Or is there something else?